lms/djangoapps/grades/docs/decisions/0004-persistent-grade-overrides.rst

```diff
@@ -0,0 +1,31 @@
+Persistent Subsection Grade Overrides
```

Comment:
Thanks a ton for investing the time to backfill these grades decisions.  Curious on approx how long that took.

Note that they are missing the `Consequences` section, as recommended in [Nygard's post](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).  Consider in which decision-docs that section would be useful - highlighting the repercussions of the decisions.

Reply:
I don't think it took more than an hour.  Thanks for the pointer, I wasn't aware of the `Consequences` section, but it's a good idea.  I'll add it at least to the grade freezing doc.